### ISSUE TITLE
Crash on with 1.1.0 on server

### ISSUE THREAD

**Issue by 123FLO321** [Christofmeg/JustEnoughBreeding#12]
```
	Caused by 0: java.lang.NoClassDefFoundError: net/minecraft/client/renderer/MultiBufferSource
		at com.christofmeg.justenoughbreeding.config.integrated.MinecraftIntegration.<clinit>(MinecraftIntegration.java:32) ~[justenoughbreeding-forge-1.20.x-1.1.0.jar%23369!/:1.1.0] {re:classloading}
		at com.christofmeg.justenoughbreeding.config.JEBIntegration.init(JEBIntegration.java:12) ~[justenoughbreeding-forge-1.20.x-1.1.0.jar%23369!/:1.1.0] {re:classloading}
		at com.christofmeg.justenoughbreeding.JustEnoughBreeding.<init>(JustEnoughBreeding.java:15) ~[justenoughbreeding-forge-1.20.x-1.1.0.jar%23369!/:1.1.0] {re:classloading}
		at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?] {}
		at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source) ~[?:?] {}
		at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source) ~[?:?] {}
		at java.lang.reflect.Constructor.newInstanceWithCaller(Unknown Source) ~[?:?] {}
		at java.lang.reflect.Constructor.newInstance(Unknown Source) ~[?:?] {}
		at net.minecraftforge.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:70) ~[javafmllanguage-1.20.1-47.2.20.jar%23473!/:?] {}
		at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$4(ModContainer.java:124) ~[fmlcore-1.20.1-47.2.20.jar%23472!/:?] {}
		at java.util.concurrent.CompletableFuture$AsyncRun.run(Unknown Source) ~[?:?] {}
		at java.util.concurrent.CompletableFuture$AsyncRun.exec(Unknown Source) ~[?:?] {}
		at java.util.concurrent.ForkJoinTask.doExec(Unknown Source) ~[?:?] {}
		at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(Unknown Source) ~[?:?] {}
		at java.util.concurrent.ForkJoinPool.scan(Unknown Source) ~[?:?] {re:mixin,re:computing_frames}
		at java.util.concurrent.ForkJoinPool.runWorker(Unknown Source) ~[?:?] {re:mixin,re:computing_frames}
		at java.util.concurrent.ForkJoinWorkerThread.run(Unknown Source) ~[?:?] {re:mixin}
```
[crash.txt](https://github.com/Christofmeg/JustEnoughBreeding/files/14807367/crash.txt)

**Comment by Christofmeg** [comment]
Thanks, I'll fix this as soon as I can!

**Comment by Christofmeg** [comment]
Fixed in version 1.1.1 on 1.20.x
